src/apiCalls/configuration/getServerConfigurationForEnvironment.ts

```diff
@@ -3,10 +3,12 @@ import { EnvironmentsEnum, NetworkType } from 'types';
 import { getServerConfiguration } from './getServerConfiguration';
 
 export async function getServerConfigurationForEnvironment(
-  environment: EnvironmentsEnum
+  environment: EnvironmentsEnum,
+  apiAddress?: string
```

Comment:
this would ideally be called options and be 
```
options?: {
  apiAddress?: string
}
```

this is to make it extensible in the future